myRental/app.js

```diff
@@ -0,0 +1,37 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'ejs');
+
+// uncomment after placing your favicon in /public
+//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, '../../MyRentalWeb/build')));
+
+app.use(function(err, req, res, next) {
+    // set locals, only providing error in development
```

Comment:
Instead of adding these inline comments, to make the code more readable, you can make this a function that is called here instead. That way we can tell what is supposed to be going on without the comments.